deployments/helm/nsm-monitoring/templates/skydive.tpl

```diff
@@ -81,9 +81,10 @@ spec:
         app: skydive
         tier: analyzer
     spec:
+      serviceAccount: nsmgr-acc
       containers:
         - name: skydive-analyzer
-          image: skydive/skydive:0.23.0
+          image: matrohon/skydive:latest
```

Comment:
Why? this is unofficial image, shall we rely on it?

Reply:
a PR is ongoing on skydive : https://github.com/skydive-project/skydive/pull/2076

for the time being, skydive is not compatible anymore with latest NSM